src/Serilog.Sinks.OpenTelemetry/OpenTelemetryLoggerConfigurationExtensions.cs

```diff
@@ -33,23 +34,30 @@ public static class OpenTelemetryLoggerConfigurationExtensions
 #else
         null;
 #endif
-    
+
     /// <summary>
     /// Send log events to an OTLP exporter.
     /// </summary>
     /// <param name="loggerSinkConfiguration">
     /// The `WriteTo` configuration object.
     /// </param>
     /// <param name="configure">The configuration callback.</param>
+    /// <param name="includeEnvironment">If true the configuration will be overridden with values from OTLP Exporter Configuration environment variables(https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/).</param>
     public static LoggerConfiguration OpenTelemetry(
         this LoggerSinkConfiguration loggerSinkConfiguration,
-        Action<BatchedOpenTelemetrySinkOptions> configure)
+        Action<BatchedOpenTelemetrySinkOptions> configure,
+        bool includeEnvironment = true)
```

Comment:
If the variables are going to be considered by default, the parameter name and meaning should probably be inverted, e.g.: `ignoreEnvironment = false` (Serilog APIs follow an "optional booleans default to false" convention wherever possible.)